engine/src/flutter/lib/web_ui/test/canvaskit/surface_test.dart

```diff
@@ -213,21 +213,22 @@ void testMain() {
 
         // Emulate WebGL context loss.
         final DomOffscreenCanvas canvas = surface.debugGetOffscreenCanvas()!;
-        final Object ctx = canvas.getContext('webgl2')!;
-        final Object loseContextExtension = js_util.callMethod(ctx, 'getExtension', <String>[
-          'WEBGL_lose_context',
-        ]);
-        js_util.callMethod<void>(loseContextExtension, 'loseContext', const <void>[]);
+        final JSObject ctx = canvas.getContext('webgl2')! as JSObject;
+        final JSObject loseContextExtension = ctx.callMethod<JSObject>(
+          'getExtension'.toJS,
+          'WEBGL_lose_context'.toJS,
+        );
+        loseContextExtension.callMethod('loseContext'.toJS);
 
         // Pump a timer to allow the "lose context" event to propagate.
         await Future<void>.delayed(Duration.zero);
         // We don't create a new GL context until the context is restored.
         expect(surface.debugContextLost, isTrue);
-        final bool isContextLost = js_util.callMethod<bool>(ctx, 'isContextLost', const <void>[]);
+        final bool isContextLost = ctx.callMethod<JSBoolean>('isContextLost'.toJS).toDart;
```

Comment:
[`isContextLost`](https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/isContextLost) can also be added to the `WebGLContext` type.